vnext/Microsoft.ReactNative/Modules/DevSettingsModule.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "DevSettingsModule.h"
+
+namespace Microsoft::ReactNative {
+
+auto DevSettingsModule::getConstants() -> std::map<std::string, folly::dynamic> {
+  return {};
+}
+
+const char *DevSettingsModule::name = "DevSettings";
+std::string DevSettingsModule::getName() {
+  return name;
+}
+
+DevSettingsModule::DevSettingsModule(Mso::VoidFunctor &&reload) : m_reload(std::move(reload)) {}
+
+void DevSettingsModule::reload() {
+  m_reload();
+}
+void DevSettingsModule::setHotLoadingEnabled(bool /*isHotLoadingEnabled*/) {}
```

Comment:
>setHotLoadingEnabled [](start = 24, length = 20)

Are they to be implemented later?

Reply:
Yeah, I should probably do something like assert, since these are not implemented.  They are not called by any of the standard RN code, so I dont think they are really needed.